etl/steps/data/garden/demography/2023-03-31/population/population.meta.yml

```diff
@@ -59,7 +59,7 @@ tables:
 
     variables:
       population:
-        title: Population
+        title: Total population (Gapminder, HYDE & UN)
```

Comment:
Unsure about "hardcoding" source information to the indicator title

Reply:
My bad, putting back `Population`.